prov/efa/src/rdm/rxr_ep.h

```diff
@@ -245,6 +245,8 @@ struct rxr_ep {
 	struct fid_peer_srx peer_srx; /* support sharing receive context with peer providers */
 	bool cuda_api_permitted; /**< whether end point is permitted to call CUDA API */
 	struct fi_info *user_info; /**< fi_info passed by user when calling fi_endpoint */
+	bool sendrecv_in_order_aligned_128_bytes; /**< whether to support in order send/recv of each aligned 128 bytes memory region */
```

Comment:
what is the `/**<` syntax?  Is that doxygen?

Reply:
Yes. that is doxygen.